### ISSUE TITLE
加速yolov8-sahi推理速度

### ISSUE THREAD

**Issue by ZHIZIHUABU** [dceluis/sahi_batched#1]

**Comment by ZHIZIHUABU** [comment]
感谢你的工作，我把你的代码替换进去后，推理速度有提升，请问你是对切分的子图堆叠成batch输入网络来加速的吗？我看在SAHI库中还有包含batch_num等后处理加速操作，在你的代码中有体现吗？ultralytics库的代码跳转实在太乱了，跳转找不到对应代码。

**Comment by dceluis** [comment]
Hello, sorry for the late reply.

Indeed the SAHI code is quite hard to follow and this repository is too because I mostly copied and pasted their inference code.

The main addition of this library is this line:
https://github.com/dceluis/sahi_batched/blob/5b050bcb7e12ed086e0d10b38edaa0ef555ac5f3/sahi_batched/main.py#L458
where, instead of passing a single image, we send the whole list of slices. YoloV8 supports sliced inference from a list of images but the original SAHI does not make use of this feature.

I did not modify or include any post-processing code from the original SAHI.
